distributor-api-v1/pricing.md

```diff
@@ -0,0 +1,72 @@
+# Pricing
+
+* **Net price** is an amount without taxes.
+* **Gross price** is a net price + taxes.
+
+In order to use pricing in the Distributor, the API returns the [Amount](Amount) object as part of responses.
+[Amount](Amount) represents a structure that holds gross price, net price and tax values.
+
+The [Amount](Amount) has following structure:
+
+```javascript
+{
+    "USD":
+        {
+            "GrossValue": 100.00,
+            "NetValue": 93.46,
+            "TaxValues": [
+                {
+                    "TaxRateCode": "DE-R",
+                    "Value": 6.54
+                }
+            ]
+        }
+}
+```
+
+### Amount
+
+| Property | Type | Description |
+| :--- | :--- | :--- |
+| `GrossValue` | Number | Gross value  |
```

Comment:
Missing dots at the ends of sentences. Why isn't it described here what gross is and what net is?